README.md

```diff
@@ -124,3 +124,11 @@ Not right now. We might add this feature in future, but for now, consider either
 
 - Adding your dependency to the package set if possible, or
 - Creating your own custom package set
+
+### Why are my changes not updated in my package set?
+
+Package sets are cached in `.psc-package`. If you are changing a
```

Comment:
When you say changing a package set, do you mean manually? Could you please clarify that? Also, could you please move the command into its own block-level element?


Thanks!

Reply:
Also, maybe point out that we cache based on Git tag names?